lacework-agent/values.schema.json

```diff
@@ -201,6 +207,16 @@
                         }
                     ]
                 },
+                "anonymizeIncoming": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": ["netmask"],
+                    "properties": {
+                        "netmask": {
+                            "type": ["string", "null"]
+                        }
+                    }
+                },
```

Comment:
@jprakash-lw When adding a new required property, please call-out in the release notes.

By the way, why is netmask required? The helm template treats it as optional.